modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -30,6 +30,7 @@
 #include <ghoul/ext/assimp/contrib/zip/src/zip.h>
 #include <ghoul/logging/logmanager.h>
 #include <unordered_map>
+#include <fstream>
```

Comment:
```suggestion
#include <fstream>
#include <unordered_map>
```

We order the includes alphabetically, with the `openspace` ones in a block first, followed by the `ghoul` ones, followed by the remaining